lib/collection/widgets/fourth_pass_food_item_card.dart

```diff
@@ -25,6 +31,8 @@ class FourthPassFoodItemCard extends StatelessWidget {
                 foodItem.timePeriod.value),
             TextFormField(
               readOnly: true,
+              // TODO add snackbar confirmation and scroll in the given page
+              onTap: () => onSelectedScreenChanged!(2),
```

Comment:
With relation to previous conversations about named strings, is the plan to eventually replace these indices with readable strings? (in the sense that '2' as a selected screen is vague without context)

Reply:
True, done 😊.